packages/client/components/BottomControlBarReady.tsx

```diff
@@ -0,0 +1,133 @@
+import styled from '@emotion/styled'
+import graphql from 'babel-plugin-relay/macro'
+import useAtmosphere from 'hooks/useAtmosphere'
+import {TransitionStatus} from 'hooks/useTransition'
+import FlagReadyToAdvanceMutation from 'mutations/FlagReadyToAdvanceMutation'
+import React from 'react'
+import {createFragmentContainer} from 'react-relay'
+import {PALETTE} from 'styles/paletteV2'
+import {BezierCurve} from 'types/constEnums'
+import BottomNavControl from './BottomNavControl'
+import BottomNavIconLabel from './BottomNavIconLabel'
+import Icon from './Icon'
+import BottomControlBarProgress from './BottomControlBarProgress'
+import handleRightArrow from 'utils/handleRightArrow'
+import useGotoNext from 'hooks/useGotoNext'
+import isDemoRoute from 'utils/isDemoRoute'
+import {BottomControlBarReady_meeting} from '__generated__/BottomControlBarReady_meeting.graphql'
+import {NewMeetingPhaseTypeEnum} from 'types/graphql'
+
+interface Props {
+  isDemoStageComplete?: boolean
+  meeting: BottomControlBarReady_meeting
+  status: TransitionStatus
+  onTransitionEnd: () => void
+  handleGotoNext: ReturnType<typeof useGotoNext>
+}
+
+const CheckIcon = styled(Icon)<{progress: number; isNext: boolean}>(({progress, isNext}) => ({
+  color: isNext
+    ? progress === 1
+      ? PALETTE.TEXT_GREEN
+      : PALETTE.EMPHASIS_WARM
+    : progress > 0
+    ? PALETTE.TEXT_GREEN
+    : PALETTE.TEXT_GRAY,
+  fontSize: 24,
+  fontWeight: 600,
+  height: 24,
+  transformOrigin: '0 0',
+  // 20px to 16 = 0.75
+  transform: progress > 0 ? `scale(0.75)translate(4px, 4px)` : undefined,
+  transition: `transform 100ms ${BezierCurve.DECELERATE}`
+}))
+
+const BottomControlBarReady = (props: Props) => {
+  const {handleGotoNext, isDemoStageComplete, meeting, onTransitionEnd, status} = props
+  const {id: meetingId, facilitatorUserId, localStage, meetingMembers, reflectionGroups} = meeting
+  const {id: stageId, isViewerReady, phaseType} = localStage
+  const {gotoNext, ref} = handleGotoNext
+  const activeCount = meetingMembers.filter((member) => member.isCheckedIn).length
+  const atmosphere = useAtmosphere()
+  const {viewerId} = atmosphere
+  const isFacilitating = facilitatorUserId === viewerId
+  const readyCount = localStage.readyCount || 0
+  const progress = readyCount / Math.max(1, activeCount - 1)
+  const onClick = () => {
+    if (isFacilitating) {
+      gotoNext()
+    } else {
+      FlagReadyToAdvanceMutation(atmosphere, {isReady: !isViewerReady, meetingId, stageId})
+    }
+  }
+  const onKeyDown = isFacilitating
+    ? handleRightArrow(() => {
+        gotoNext()
+      })
+    : undefined
+  const icon = isFacilitating ? 'arrow_forward' : 'check'
+  const label = isFacilitating ? 'Next' : 'Ready'
+  const getDisabled = () => {
```

Comment:
Oh god, I feel so embarrassed now. I get the logic. Disabled doesn't have anything to do with how many people said they are ready. There was a special case during the Reflect phase. This makes sense. Ugh. I feel so silly.